autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectColumn.css

```diff
@@ -0,0 +1,36 @@
+.container {
+    width: 20%;
+    height: 100%;
+    padding-top: 1%;
+    padding-left: 1%;
+}
+
+/* Having this wrapper lets us fix the top & bottom of the .course-select-column of the page
+   so the picker scrolls "under" the button while still being within the bounds of the
+   page
+*/
+.column-wrapper {
+    position: relative;
+    width: 100%;
+    height: 93%;
+}
+
+.course-select-column {
+    position: absolute;
+    top: 0px;
+    bottom: 0px;
+    overflow-y: auto;
+    width: 100%;
+
+    padding-top: 1%;
+    padding-left: 1%;
+}
+
+.row {
+    padding-bottom: 10px;
+    padding-right: 5%;

```

Comment:
I also think the padding-right should probably be on the entire column, rather than the individual cards so I'm gonna change it

Reply:
So in regards to dynamically determining if there's a scroll bar: I have it implemented, but it's always "one step behind", so it won't actually detect the scroll bar until I add a course card when there is a scroll bar. I believe this is because I'm loading in the previous rendering's `course-select-column` element as it do it before we returning the new components to be rendered.

Reply:
Changed the padding to 2% though

Reply:
Resolving this since #175 was merged